### DISCUSSION TITLE
Way to use second GPU on my system instead of main one

### DISCUSSION THREAD

**Discussion by TeamBoV** [azuritecoin/OnnxDiffusersUI#66] [Q&A]
Hello. I have 2 RX 480 8GB on my system, is there a way to achieve this (via a config file or argument, for example) so my screen doesn't lag and I can keep using my computer with my main GPU for other tasks? Everything else is working fine as far as I can tell, so thank you!

**Comment by azuritecoin** [comment]
There might be a way... Requires some investigation on how to point the program to use the second GPU
